src/script/interpreter.cpp

```diff
@@ -1254,8 +1253,12 @@ class CTransactionSignatureSerializer
     const CScript& scriptCode; //!< output script being consumed
     const unsigned int nIn;    //!< input index of txTo being signed
     const bool fAnyoneCanPay;  //!< whether the hashtype has the SIGHASH_ANYONECANPAY flag set
+    // Temporary workaround for a clang-tidy bug.
```

Comment:
Given the upstream fix has landed (and was possibly backported?), this workaround should either be removed entirely, or updated to indicate the version of Clang which this was fixed in.